cl/lib/search_utils.py

```diff
@@ -864,7 +917,17 @@ def cleanup_main_query(query_string: str) -> str:
         else:
             cleaned_items.append(f'"{item}"')
 
-    return "".join(cleaned_items)
+    cleaned_query = "".join(cleaned_items)
```

Comment:
Please don't forget to update the docstring above to include this.

Reply:
Sure, I've updated the docstrings in this method.